pages/peopleloc.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">People Associated with the Witch Trials: Residence</h1>
```

Comment:
Lets merge it in then remove. Because all of the pages are chained off each other, it is cleaner to merge it, then remove at the end in a separate PR. Hope that makes sense

Reply:
Marking this as resolved and will approve but not merge yet as the people branch isn't ready